Rakefile

```diff
@@ -1,10 +1,8 @@
-require "bundler/gem_tasks"
-require "rake/testtask"
-
-Rake::TestTask.new(:test) do |t|
-  t.libs << "test"
-  t.libs << "lib"
-  t.test_files = FileList["test/**/*_test.rb"]
+begin
+  require "rspec/core/rake_task"
+  RSpec::Core::RakeTask.new(:spec)
+rescue LoadError
```

Comment:
I think we should raise instead of rescuing and acting like nothing was wrong. That's also good for CI since raising would exit with an erroneous status.

Reply:
It completely makes sense. As your review, I just simply need to remove the begin block since it will raise the exception automatically.